aiosmtpd/tests/test_smtp.py

```diff
@@ -241,6 +269,7 @@ class TestProtocol(unittest.TestCase):
     def setUp(self):
         self.transport = Mock()
         self.transport.write = self._write
+        self.transport.get_extra_info.return_value = "MockedPeer"
```

Comment:
This is so that in the TESTLOGFILE, the log lines will be tidy. `MockedPeer` instead of `<Mock object bla bla bla id="12345567" etc yadda yadda>`